packages/authorization-process/test/createConsumerClient.test.ts

```diff
@@ -0,0 +1,76 @@
+import { describe, it, vi, beforeAll, afterAll, expect } from "vitest";
+import {
+  Client,
+  ClientAddedV2,
+  UserId,
+  clientKind,
+  generateId,
+  toClientV2,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  decodeProtobufPayload,
+  readLastEventByStreamId,
+} from "pagopa-interop-commons-test/index.js";
+import { ApiClientSeed } from "../src/model/domain/models.js";
+import { authorizationService, postgresDB } from "./utils.js";
+
+describe("createConsumerClient", () => {
+  const organizationId = generateId();
+
+  beforeAll(async () => {
+    vi.useFakeTimers();
+    vi.setSystemTime(new Date());
+  });
+
+  afterAll(() => {
+    vi.useRealTimers();
+  });
+
+  const clientSeed: ApiClientSeed = {
+    name: "Seed name",
+    description: "Description",
+    members: [organizationId],
+  };
+  it("should write on event-store for the creation of a consumer client", async () => {
+    const { client } = await authorizationService.createConsumerClient(
+      clientSeed,
+      unsafeBrandId(organizationId),
+      generateId(),
+      genericLogger
+    );
```

Comment:
I think it's better to always use named parameters in all the endpoints, so that we avoid situations like this, where it's unclear what `generateId` is used for.